hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/protocol/DatanodeDetails.java

```diff
@@ -619,11 +629,11 @@ public void setCurrentVersion(int currentVersion) {
 
   @Override
   public String toString() {
-    return uuidString + "(" + hostName + "/" + ipAddress + ")";
+    return id + "(" + hostName + "/" + ipAddress + ")";
```

Comment:
Can we store `uuidString` in `DatanodeID` and return it in `toString`, to avoid repeated conversion?